### PR TITLE
Mc nv homepage design

### PR DESCRIPTION
**Pull request by mcon85**
changed logo, changed padding on logo, changed color of nav background

### CODE REVIEW

**Comment by mcon85** [comment]
Still trying to figure out how to get rid of the top level comment hound issue

File: Gemfile.lock [hunk 1]
```diff
@@ -74,12 +74,12 @@ GEM
     factory_girl_rails (4.7.0)
       factory_girl (~> 4.7.0)
       railties (>= 3.0.0)
+    font-awesome-rails (4.6.3.1)
```
Comment (jraenorris) [inline Gemfile.lock]: Look in PR #24 for a comment regarding these changes

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -2,3 +2,7 @@
 *= require foundation_and_overrides
 *= require font-awesome
 */
+
+.menu .logo {
```
Comment (jraenorris) [inline app/assets/stylesheets/application.scss]: Pull this out into its own stylesheet and require that file in this one

File: app/controllers/homepage_controller.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class HomepageController < ApplicationController
```
Comment (jraenorris) [inline app/controllers/homepage_controller.rb]: You keep getting this hound violation. Update your .yml file if you need to

File: config/routes.rb [hunk 4]
```diff
@@ -3,6 +3,7 @@
 
   root 'venues#index'
 
+  resources :homepage, only: [:index]
```
Comment (jraenorris) [inline config/routes.rb]: why is it called homepage if it's not actually your home page (root)?

File: app/controllers/homepage_controller.rb (hunk 3 above)
Comment (nvanselow) [inline app/controllers/homepage_controller.rb]: I'm not sure why the comment is still here. Below it says " No violations found. Woof!". I think this won't show up in future ones now that the hound config has been merged.

Comment (jraenorris) [inline app/controllers/homepage_controller.rb]: Ahh okay

**Pull request merged**